packages/ui/origin-ui-core/src/components/organization/organizationMenu.ts

```diff
@@ -9,19 +9,20 @@ import { OrganizationForm } from './OrganizationForm';
 import { OrganizationTable } from './OrganizationTable';
 import { OrganizationView } from './OrganizationView';
 import { fromUsersSelectors } from '../../features';
+import { useOriginConfiguration } from '../../utils/configuration';
 
 interface IOrganizationMenuItem {
     key: string;
     label: string;
     component: React.ReactType;
     show: boolean;
-    feature?: OriginFeature;
```

Comment:
It wasn't used and IMO at this point it only duplicates logic `show` is doing in slightly more declarative way. However, because of `?` handling this would be more complex, so it's better to remove it for now